src/software/amazon/ion/impl/lite/IonSystemLite.java

```diff
@@ -194,6 +193,11 @@ public SymbolTable getSystemSymbolTable(String ionVersionId)
     public Iterator<IonValue> iterate(byte[] ionData)
```

Comment:
it is not obvious that this method is problematic;  please add a comment indicating why it's deprecated

Reply:
The comment is in the interface javadoc but will add a note here as well.